packages/plugin-npm-cli/sources/commands/npm/audit.ts

```diff
@@ -69,6 +79,14 @@ export default class AuditCommand extends BaseCommand {
     validator: t.isEnum(npmAuditTypes.Severity),
   });
 
+  excludes = Option.Array(`--exclude`, [], {
+    description: `Packages to exclude from audit`,
+  });
```

Comment:
To match the other commands this should probably be glob patterns instead.
https://github.com/yarnpkg/berry/blob/b8d16768d9397d2d814163b0dc70dbf3e5074fab/packages/plugin-workspace-tools/sources/commands/foreach.ts#L102-L104

Reply:
Have pushed a change to do globs